packages/edit-site/src/store/selectors.js

```diff
@@ -64,6 +65,22 @@ export const getCanUserCreateMedia = createRegistrySelector( ( select ) => () =>
 	select( coreDataStore ).canUser( 'create', 'media' )
 );
 
+/**
+ * Returns any available Reusable blocks.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {Array} The available reusable blocks.
+ */
+export const getReusableBlocks = createRegistrySelector( ( select ) => () => {
```

Comment:
Is this selector really needed? Any reason for not calling the underlying selector directly?

Reply:
I guess we could keep it but not expose it.

Reply:
```suggestion
const getReusableBlocks = createRegistrySelector( ( select ) => () => {
```

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/36511/commits/c11a2490ddf4b0af3a78e68d87af4a8b8345f171

Reply:
Actually this doesn't work and introduces an error. I think this is to do with this selector using `createRegistrySelector`. Unless it's exported we end up in an error state.